support/rally.js

```diff
@@ -84,15 +85,31 @@ export default class Rally {
         await browser.runtime.sendMessage(CORE_ADDON_ID, msg, {});
 
       if (!response
-          || response.type !== "core-check-response"
-          || response.data.enrolled !== true) {
+        || response.type !== "core-check-response"
+        || response.data.enrolled !== true) {
```

Comment:
Mh, ok, what's up with these changes here? Why is your IDE unhappy with our rules? Do we need to change our rules? :)

Reply:
I suspect our eslint isn't strict enough, it seems fine with or without these changes. Not sure why VSCode isn't using the same exact config if that's the case though?

Anyway happy to just remove these :) I filed an issue on the linting stuff.